### DISCUSSION TITLE
npm install error

### DISCUSSION THREAD

**Discussion by akhtarmo** [sibiraj-s/ngx-editor#442] [Q&A] [answered]
I am getting an error while installing the ngx-editor. My environment details as below:
Angular CLI: 14.0.6
Node: 16.16.0
Package Manager: npm 8.14.0
OS: win32 x64
Angular:
...
Package                      Version
------------------------------------------------------
@angular-devkit/architect    0.1400.6 (cli-only)
@angular-devkit/core         14.0.6 (cli-only)
@angular-devkit/schematics   14.0.6 (cli-only)
@schematics/angular          14.0.6 (cli-only)

The error i am getting as below:
npm ERR! Conflicting peer dependency: @angular/common@14.2.11
npm ERR! node_modules/@angular/common
npm ERR!   peer @angular/common@">=14.1.0" from ngx-editor@15.0.1
npm ERR!   node_modules/ngx-editor
npm ERR!     ngx-editor@"*" from the root project

Please help

**Comment by sibiraj-s** [comment, marked as answer]
Hi. The minimum required version for the editor is v14.1.0 as the editor uses some features that are available only on 14.1.0 and later.

[a comment by an automated account is not shown]
